File: 0997-find-the-town-judge/0997-find-the-town-judge.py
class Solution:
    def findJudge(self, n: int, trust: List[List[int]]) -> int:
        data = [0]*(n+1)
        
        for a,b in trust:
            data[a] -= 1
            data[b] += 1
        
        for i in range(1, n+1):
            if data[i]==n-1:
                return i
        return -1

File: 0997-find-the-town-judge/0997-find-the-town-judge.cpp
class Solution {
public:
    int findJudge(int n, vector<vector<int>>& trust) {
        vector<int> trusted(n+1, 0);
        for(int i=0; i<trust.size(); i++){
            trusted[trust[i][0]]--;
            trusted[trust[i][1]]++;
        }
        
        for(int i=1; i<n+1; i++){
            if(trusted[i]==n-1)
                return i;
        }
        return -1;
    }
};